pkg/printers/internalversion/printers.go

```diff
@@ -876,8 +901,18 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 		}
 		break
 	}
-	if !initializing {
-		restarts = 0
+
+	hasInitialized := false
```

Comment:
Why aren’t we just using the Initialized condition for this check?

Reply:
I just found that the initialized condition is not aligned with the sidecar containers. will fix it.

Reply:
fixed.